### ISSUE TITLE
The new version of NVEnc encodes h265 format, the video has black borders

### ISSUE THREAD

**Issue by shookgao** [rigaya/NVEnc#253]
Hi rigaya: 
1、Happens when encoding is h265, h264 does not;
2、It will be seen when using the potplayer player, other players will not;
3、The new version of NVEnc will, the old version will not (before 5.08);
4、Sometimes the black border is on the bottom, sometimes on the left and bottom, sometimes on the left, bottom and right;
5、I used the --crop parameter when encoding.
`nvencc --avhw --profile main10 --preset quality --codec hevc --crop x,y,x,y --audio-copy 1 --sub-copy 1 --chapter-copy -i i.mkv -o o.mkv`
what happened? Not every movie will have problems, probably the probability is 4/5

**Comment by rigaya** [comment]
Unfortunately, even though I've checked with your options, I have not been able to reproduce your problem with potplayer and other players. The output resolution checked by MediaInfo also seems to be fine as the result of --crop.

I've checked changes between 5.08→5.09, but don't seem to have related changes. 

> It will be seen when using the potplayer player, other players will not;

If the other players shows fine, the output might be fine.
